pkg/cloudprovider/providers/openstack/openstack.go

```diff
@@ -412,6 +415,10 @@ func ReadConfig(config io.Reader) (Config, error) {
 		cfg.Metadata.SearchOrder = fmt.Sprintf("%s,%s", metadata.ConfigDriveID, metadata.MetadataID)
 	}
 
+	if !util.Contains(supportedLBProvider, cfg.LoadBalancer.LBProvider) {
```

Comment:
the commit msg said it's must fit and `Contains` are used here, if it's a typo such as `octaviaa` then still pass this test?

Reply:
@jichenjc `Contains searches if a string list contains the given string or not.`

Reply:
my bad ...

Reply:
:-) all good